pkgs/top-level/python-packages.nix

```diff
@@ -19678,10 +19678,10 @@ in modules // {
     # to packages like carbon (http://stackoverflow.com/questions/19894708/cant-start-carbon-12-04-python-error-importerror-cannot-import-name-daem)
     disabled = isPy3k;
 
-    name = "Twisted-11.1.0";
+    name = "Twisted-15.5.0";
     src = pkgs.fetchurl {
       url = "https://pypi.python.org/packages/source/T/Twisted/${name}.tar.bz2";
-      sha256 = "05agfp17cndhv2w0p559lvknl7nv0xqkg10apc47fm53m8llbfvz";
+      sha256 = "0zy18lcrris4aaslil5k12i13k56c32hzfdv6h10kbnzl026h158";
     };
```

Comment:
Unless you tested with Nox what the effects are of this upgrade I suggest you add a` twisted_15` instead and use that. We can then later see what other packages can use this latest twisted. The same with `simplejson` since that's also a common package. Unless of course you run into collisions.